tests/acceptance/29_simulate_mode/files_promises.cf

```diff
@@ -0,0 +1,43 @@
+body common control
+{
+      inputs => { "../default.cf.sub" };
+      bundlesequence  => { default("$(this.promise_filename)") };
+      version => "1.0";
+}
+
+bundle agent test
+{
+  meta:
+    "test_soft_fail" string => "(solaris|aix|hpux)|(debian_7.i686)",
+      meta => { "ENT-6537", "ENT-6540" };
+      # ENT-6537 debian 7 dirent has extra ',' entry
+      # ENT-6540 exotics fail to delete chroot
+
+    "description" -> { "ENT-5301" }
+      string => "Test that files promises in --simulate=manifest mode produce proper output and only make changes in chroot";
+
+  commands:
+    # add --verbose here and look at the .actual log for debugging sub policy runs
+    "$(sys.cf_agent) -Kf $(this.promise_filename).sub --simulate=manifest > $(this.promise_filename).temp 2>&1"
+      contain => in_shell,
+      handle => "agent_run_complete",
+      comment => "Run sub policy in manifest mode and capture output to $(this.promise_filename).actual file.";
+    "$(G.sed)"
+      args => " \
+-e 's,$(sys.workdir),WORKDIR,' \
+-e 's/[0-9]*\.changes/PID.changes/' \
+-e 's/[0-9]\{4\}-[0-9]\{2\}-[0-9]\{2\}[[:blank:]][0-9]\{2\}:[0-9]\{2\}:[0-9]\{2\}[[:blank:]][-+][0-9]\{4\}/TIMESTAMP/' \
+-e 's/Size: [0-9]*/Size: SIZE/' \
+$(this.promise_filename).temp > $(this.promise_filename).actual",
+      contain => in_shell,
+      handle => "warning_line_normalized",
```

Comment:
comment => "Re-write volatile information (time, size, etc ....) so that things can be tested ...."